### PR TITLE
Fixing minor code quality issues

### PR DESCRIPTION
**Pull request by joshcanhelp**
Ran through Code Inspector in PhpStorm and PHP CodeSniffer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,6 @@
 [![Build Status](https://travis-ci.org/auth0/auth0-PHP.png)](https://travis-ci.org/auth0/auth0-PHP)
 [![Code Climate](https://codeclimate.com/github/auth0/auth0-PHP/badges/gpa.svg)](https://codeclimate.com/github/auth0/auth0-PHP)
 [![Test Coverage](https://codeclimate.com/github/auth0/auth0-PHP/badges/coverage.svg)](https://codeclimate.com/github/auth0/auth0-PHP/coverage)
-[![Dependencies](https://www.versioneye.com/php/auth0:auth0-php/badge.svg)](https://www.versioneye.com/php/auth0:auth0-php)
```
Comment (joshcanhelp) [inline README.md]: Shut down - https://blog.versioneye.com/2017/10/19/versioneye-sunset-process/

File: src/API/Authentication.php [hunk 2]
```diff
@@ -1,4 +1,13 @@
 <?php
```
Comment (joshcanhelp) [inline src/API/Authentication.php]: Summary of changes:

- Spacing and indenting (automatically with `phpcbf`)
- Docblocks
- Private properties get a `_` prefix
- Line length = 85

File: tests/API/BasicCrudTest.php [hunk 3]
```diff
@@ -49,7 +49,7 @@ public function testAll() {
 
         $this->afterCreate($created);
 
-        $client3 = $client->update($this->getId($created), $this->getUpdateBody());
```
Comment (joshcanhelp) [inline tests/API/BasicCrudTest.php]: Not used

File: README.md [hunk 4]
```diff
@@ -4,8 +4,6 @@
...
-[![HHVM Status](http://hhvm.h4cc.de/badge/auth0/auth0-php.svg)](http://hhvm.h4cc.de/package/auth0/auth0-php)
```
Comment (joshcanhelp) [inline README.md]: Not workin'

File: tests/API/Management/BlacklistsTest.php [hunk 5]
```diff
@@ -6,6 +6,8 @@
 
 class BlacklistsTest extends ApiTests {
 
+    private $domain;
```
Comment (cocojoe) [inline tests/API/Management/BlacklistsTest.php]: How come no `_` for this private property, or tests out of scope in this refactor?

Comment (joshcanhelp) [reply tests/API/Management/BlacklistsTest.php]: @cocojoe - I ran the `phpcs/phpcbf` against all the files and the list was so long that my terminal couldn't show it all 😆 so I just used it on `src/API/Authentication.php` (inspected the rest) to see how much work as involved. Answer is: a lot but I think it's worth it. I would like to see 5.x.x "finished" before it's archived and I see this as part of that.

**Comment by joshcanhelp** [comment]
@cocojoe - Approved?

**Pull request merged**